rebar.config

```diff
@@ -1,16 +1,13 @@
 {cover_enabled, true}.
 
 {erl_opts, [
-	debug_info,
-	fail_on_warning,
-    {parse_transform, lager_transform}
+	debug_info
 ]}.
 
 {deps, [
-        {nklib, ".*", {git, "https://github.com/michalwski/nklib.git", {ref, "006c1b7"}}},
+        {nklib, ".*", {git, "https://github.com/esl/nklib.git", {branch, "remove-lager"}}},
```

Comment:
Shouldn't it be changed to master?

Reply:
Yes of course :) I planned that, but I forgot to mark the PR as draft indeed.